### PR TITLE
[W5.11][F14-A4] Jordan Chong

### PR DESCRIPTION
**Pull request by jordancjq**
__Feature Enhancement:__
1. Added a new `sort` command that sorts and display the address book by alphabetical order
2. Documentation of `sort` command
3. Updated `IO` and `JUnit` test cases to reflect `sort` command

### CODE REVIEW

File: doc/UserGuide.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorting all persons : `sort`
```
Comment (pkaijun) [inline doc/UserGuide.md:56]: Simple and straight to the point of what the function is for

File: src/seedu/addressbook/commands/HelpCommand.java [hunk 2]
```diff
@@ -21,6 +21,7 @@ public CommandResult execute() {
                 + "\n" + ClearCommand.MESSAGE_USAGE
                 + "\n" + FindCommand.MESSAGE_USAGE
                 + "\n" + ListCommand.MESSAGE_USAGE
+                + "\n" + SortCommand.MESSAGE_USAGE
```
Comment (pkaijun) [inline src/seedu/addressbook/commands/HelpCommand.java:24]: Good consideration for the help menu

**Comment by pkaijun** [comment]
Overall the `sort` command is implemented well with proper documentation and test cases. Great effort for Junit too!

**Review by liuyang1213 — COMMENTED** [review]
Awesome job, all required components have been updated! Remember to use the present tense in commit message next time, for example, use "add" instead of "added".

**Pull request closed without merging**